swap_meet/item.py

```diff
@@ -1,2 +1,31 @@
 class Item:
-    pass
\ No newline at end of file
+    def __init__(self, category = "", condition=0.0):
+        self.category = category
+        self.condition = condition
+    
+    def __str__(self):
+        self.item = "Hello World!"
+        return self.item
+
+    def condition_description(self):
+        {}
+        if 0 < self.condition <= 1:
+            return "should be free tbh"
+
+        elif 1 < self.condition <= 2:
```

Comment:
Since these are elif statements, some of these comparisons are not needed! For example, on this line, you don't need `1 <` since the condition has to be greater than 1 to get to this point. (Unless you have negative numbers for condition, of course.)